views/pages/partials/tabelaPrescricaoAcolhido.ejs

```diff
@@ -32,16 +32,17 @@
                 </form>
             </td>
             <%if (tipoDoUsuario === "medica") { %>
-            <td>
-                <form action="<%= acolhido.id %>/prescricao/<%= prescricao.id %>/destroy" method="post" class="botoes">
-                    <input type="submit" value="Excluir" onclick="return confirm('Tem certeza que deseja excluir?');">
-                </form>
-            </td>
-            <td>
-                <form action="/acolhido/<%= acolhido.id%>" method="get" class="botoes">
-                    <input type="submit" value="Enviar">
-                </form>
-            </td>
+              <td>
+                  <form action="<%= acolhido.id %>/prescricao/<%= prescricao.id %>/destroy" method="post" class="botoes">
+                      <input type="submit" value="Excluir" onclick="return confirm('Tem certeza que deseja excluir?');">
+                  </form>
+              </td>
+              <td>
+                  <form action="/acolhido/<%= acolhido.id%>" method="get" class="botoes">
+                      <input type="submit" value="Enviar">
+                  </form>
+              </td>
         </tr>
-        <%}%>
-</table>
\ No newline at end of file
+            <%}%>
+    <% } %>
```

Comment:
O problema é que faltou esta última chavezinha que fecha o `for` lá de cima. :slightly_smiling_face: